src/apprt/gtk/Surface.zig

```diff
@@ -346,6 +346,9 @@ cursor: ?*c.GdkCursor = null,
 /// pass it to GTK.
 title_text: ?[:0]const u8 = null,
 
+/// The timer used to delay title updates in order to prevent flickering.
+update_title_timer: ?c.guint = null,
```

Comment:
This probably needs to be invalidated if the surface is closed.

Reply:
Added the invalidation in `deinit` function.